internal/pkg/build/files/files.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package files
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func expandPath(path string) ([]string, error) {
+	var output, stderr bytes.Buffer
+	cmd := exec.Command("/bin/sh", "-c", "echo "+path)
+	cmd.Stdout = &output
+	cmd.Stderr = &stderr
+	if err := cmd.Run(); err != nil {
+		return nil, fmt.Errorf("%s: %s", err, stderr.String())
+	}
+
+	return strings.Fields(output.String()), nil
```

Comment:
since path can expand to filenames with spaces, I don't think this works.

In principle this works:

```
#!/bin/sh

for n in "$@" ; do
    printf "$n\0"
done
```

the arguments to that script get expanded by the calling shell, and that script prints each one separated by a NULL.